geometry_msgs/Point position
nav_msgs/OccupancyGrid  map